jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/ExampleTableConstraints.java

```diff
@@ -8,6 +8,7 @@
     public static final String NAME = "name";
     public static final String DATA = "data";
     public static final String ALIAS = "contextAlias";
+    public static final String ACTUAL_VALUE = "actualValue";
```

Comment:
I have two reasons for adding new column. 
First: data doesn't makes a sence there, as actualValue better describe what you want to do. If I change it to data or something, I would prefer to changing name of second column to somethink like data2.
Second: I'm just to lazy to refactor possibly a hundred of steps in our project (as I taken step from there). But If I remove Given annotation as you suggested in another comment, I will have to refactor it anyway (or keep old method anotated with Given in project to avoid refactor at all).

Reply:
Second: ctrl shift r should do the trick if needed :slightly_smiling_face:

Reply:
It is just about mindset. If I say: "I expect my data to be verified for some value." Then it is perfectly fine reflected via `| data | expectedValue | verifier |`
If you are too lazy don't push yourself, you can always keep this functionality just in your project. On this level we have to meet framework requirements, that has to fit as much users as possible.
I vote against new column.